data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -100,7 +100,7 @@
 CENSUS_COUNTIES_COLUMNS = ["USPS", "GEOID", "NAME"]
 
 # Drop FIPS codes from map
-DROP_FIPS_CODES = ["66", "78"]
+DROP_FIPS_CODES = []
```

Comment:
i think this is the only change and it looks right to me! did you confirm that in the output, we see fips 66 and 78 with t&wd?

Reply:
i haven't! what's the simplest way to do that? when i run etl score post locally, i don't see where the output is.

Reply:
I think that runs all the way through to the downloadable files -- and if this is in the downloadable files, that is great news! I'd check here `data_pipeline/data/score/downloadable/`